soc/nordic_nrf/nrf54h/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_ARM)
+  add_subdirectory(arm)
+elseif(CONFIG_RISCV)
+  add_subdirectory(riscv)
+endif()
```

Comment:
why is this needed?

Reply:
to include the cmake file for the correct core